troubleshooting/install-nvidia-driver.md

```diff
@@ -12,6 +12,19 @@ redirect_from:
 
 You can use rpm packages from rpmfusion, or you can build the driver yourself.
 
+## Word of Caution 
+
+Proprietary (NVIDIA/AMD) drivers are known to be sometimes highly problematic, or completely unsupported. 
+AMD driver support is prebaked in the Qubes kernel (4.14) but only RADEON HD 4000-9000 give or take, NOT most of the latest cards. 
```

Comment:
vmlinuz-4.4.14-11.pvops.qubes.x86_64 
I was debating whether to put Qubes (3.2) or to be more specific with my findings. 
Yeah I'm a newbie to this. I'll try to fix the capitalization.

I worked on installing NVIDIA/AMD proprietary drivers for a good number of days now, and i found out that the kernel has prebaked support for some of the cards out there. But a lot of the newer ones like AMDGPU, and NVIDIA in general. Just don't have any prebaked modules or anything in the kernel for them. Which means you need to install a module yourself, and I haven't been able to do it, its been really tricky. I tried for the longest time with my NVIDIA 750 TI card. Just to find out the module version i installed had like an overflow bug or something with it, that the company NVIDIA hadn't resolved in that version. I now have a RADEON RX 480. And there is no prebaked support for that because its an AMDGPU. And one of the newer models. According to my research, support for AMDGPU is in the 4.5+ kernels. But i don't like messing with the kernel and I can never get a proprietary driver or kernel module install to do anything for me. 

If you tweak enough with the Kernel, and the different things. Like Xorg. Sometimes they will print a list of all the Radeon cards that are supported. That's where i got the idea for that part of it. I have a Radeon 7570 HD graphics card from 1-2 years ago. And when i plugged that in, Support was already there for it. And i could use my HDMI cable. Where as right now with my RX 480, I'm using my computers built in VGA port. 

I'll try to update the commit a bit.